### PR TITLE
support xwayland distro

### PR DESCRIPTION
**Pull request by mike-scott**
- add DISTRO config for lmp-xwayland
- adjust lmp-feature-wayland to include weston-xwayland package when x11 is selected

### CODE REVIEW

File: meta-lmp-base/conf/distro/lmp-xwayland.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require conf/distro/lmp.conf
+
+DISTRO = "lmp-xwayland"
+DISTROOVERRIDES = "lmp:lmp-wayland:lmp-xwayland"
```
[the remark this answers is not in the document]
Comment (ricardosalveti) [reply meta-lmp-base/conf/distro/lmp-xwayland.conf:4]: Yes, should be fine.

Not sure if that might end up causing issues later on, but this should be good for now.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Comment by mike-scott** [comment]
Fixing build break.

**Review by EmbeddedAndroid — APPROVED** [review]
Very nice, +1

**Pull request merged**